recipes/libnghttp2/all/conanfile.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
```

Comment:
remove this

Reply:
Many recipes have it. I think there is a proposal for the hooks to check these lines at the beginning. If there is no automation to check it or nothing is written in the FAQ I wouldn't enforce adding/removing it.

Reply:
yes, there is an issue to add an automated check and update FAQ